src/ui/MultiSelect/MultiSelect.jsx

```diff
@@ -191,15 +191,17 @@ const MultiSelect = forwardRef(
     }))
 
     const toggleItem = (selectedItem) => {
-      isItemSelected(selectedItem, selectedItems)
-        ? removeSelectedItem(selectedItem)
-        : addSelectedItem(selectedItem)
+      if (isItemSelected(selectedItem, selectedItems)) {
+        removeSelectedItem(selectedItem)
+      } else {
+        addSelectedItem(selectedItem)
+      }
```

Comment:
TS lint rules don't like ternaries being used unless a value is being set/returned so this is the way around that